drivers/sensor/sunxi-thermal.c

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright © 2017-2018 The Crust Firmware Authors.
+ * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ */
+
+#include <debug.h>
+#include <error.h>
+#include <limits.h>
+#include <mmio.h>
+#include <sensor.h>
+#include <sensor/sunxi-thermal.h>
+
+#define SENSOR_COUNT        3
+
+#define THS_0_DATA_REGISTER 0x80
+#define THS_1_DATA_REGISTER 0x84
+#define THS_2_DATA_REGISTER 0x88
+
+enum {
+	THS0,
```

Comment:
The manual should give more descriptive names: CPU? GPU?

Reply:
Hint: https://github.com/apritzel/arm-trusted-firmware/blob/allwinner-scpi/plat/sun50iw1p1/sunxi_sensors.c#L14

Reply:
![image](https://user-images.githubusercontent.com/18688003/38461044-228caf28-3a8c-11e8-8156-ebc855cad039.png)